profiles/updates/2Q-2016

```diff
@@ -0,0 +1,2 @@
+move kde-plasma/khelpcenter kde-apps/khelpcenter
+move kde-base/khelpcenter kde-apps/khelpcenter
```

Comment:
I did want to be extra careful there since there still happen to be users around that installed 4.14.3 back when it was in the overlay. Remember very strange blockers that occurred because a pkgmove was only done in tree but didn't update /var/db/pkg for overlay users...